autoendpoint/src/routers/common.rs

```diff
@@ -57,6 +57,17 @@ pub async fn handle_error(
                 error.errno(),
             );
         }
+        RouterError::GCMAuthentication => {
+            // Don't record the GCM auth error.
```

Comment:
I think this comment is inaccurate, probably want a warn! line here like the others